Assets/__Scripts/UI/SongInfoEditUI.cs

```diff
@@ -542,6 +542,8 @@ private void HandleEditMapButtonPressed(int r)
                     .BeatmapCharacteristicName;
                 Settings.Instance.LastLoadedDiff = BeatSaberSongContainer.Instance.DifficultyData.Difficulty;
                 BeatSaberSongContainer.Instance.Map = map;
+                Settings.Instance.Load_MapV3 = map.Version[0] == '3';
+                if (Settings.Instance.Load_MapV3) Settings.Instance.GridTransparency = 0.75f;
```

Comment:
No.

`Settings.Instance` will always be loaded from, and saved to, a file on disk. You are effectively changing a setting without user consent. I would much rather leave the `GridTransparency` setting untouched and have the user change it themselves if they have vision concerns.

Reply:
One of my concern is that when a user sets Load_MapV3 to true and opens a v2 map, he will ge nothing because it's not the same json format. However if he misclicks to save or is auto-saved then previous notes are all overwritten and gone, which may not be the intention..

Reply:
To clarify, my problem is not that you are modifying `Load_MapV3`, although I'm unsure if that has to be a setting to begin with. The problem I have is with modifying `GridTransparency`, since that is a setting which is directly reflected in the mapper. People will be (and already are) angry when their `GridTransparency` value is overwritten.

Reply:
Ah yes. I've removed the overwritting transparency line.